server/drivers/hd44780-4bit.c

```diff
@@ -262,9 +268,40 @@ lcdstat_HD44780_senddata(PrivateData *p, unsigned char displayID, unsigned char
  */
 void lcdstat_HD44780_backlight(PrivateData *p, unsigned char state)
 {
+	static unsigned char old_state=0;
+	unsigned char brightnessLevelVFD = 0;
+
 	p->backlight_bit = ((!p->have_backlight||state)?0:BL);
 
 	port_out(p->port, p->backlight_bit);
+
+	if (p->isVFDDisplay)
+	{
+		if (state!=old_state)
+		{
+			//for PT6314 VFD driver call init function to update BR0-BR1 registers for backlight level
+			if (p->backlightstate)
+			{
+				brightnessLevelVFD = 0x03u; //100% brightness
```

Comment:
Since you define a macro for this constant in the headerfile, please use it. Numeric constants are always a bit hard to track in mind when reading the code.